rasa/model_testing.py

```diff
@@ -127,10 +127,6 @@ def test_core_models(
     )
 
 
-# backwards compatibility
-test = rasa.test
```

Comment:
Was introduced in [this PR](https://github.com/RasaHQ/rasa/pull/8141). I think that we added that in the PR for backwards compatibility _before_ we actually renamed the file, which made this change a no-op from the beginning. If we want to be a bit extra careful we could consider adding the code below, but I don't think it's necessary:
```suggestion
def test(*args, **kwargs):
    # raise a deprecation warning that this will be removed in 4.x
    return rasa.test(*args, **kwargs)
```